spec/audited/sweeper_spec.rb

```diff
@@ -1,6 +1,12 @@
 require "spec_helper"
 
 class AuditsController < ActionController::Base
+  if Rails::VERSION::MAJOR == 4
+    before_filter :populate_user
```

Comment:
The deprecation warning was added in 5.0 - in 4.2 the use of before_filter was just ["discouraged"](http://guides.rubyonrails.org/4_2_release_notes.html#action-pack-notable-changes) so this should be fine whether AS deprecations are raised or not.